app/templates/courses/enrollments-view.jade

```diff
@@ -31,79 +31,105 @@ block content
           a.btn.btn-primary.btn-lg(href="/teachers/update-account") Upgrade to teacher account
       
     .container.m-t-5
+      .pull-right
+        span.glyphicon.glyphicon-question-sign
+        =' '
+        a#how-to-enroll-link(data-i18n="teacher.how_to_enroll")
       h3(data-i18n='teacher.enrollments')
-      h4
+      h4#enrollments-blurb
         span(data-i18n='teacher.enrollments_blurb_1')
-        span  2&ndash;8 
+        span  2&ndash;#{view.state.get('totalCourses')} 
         span(data-i18n='teacher.enrollments_blurb_2')
         
+      - var available = view.state.get('prepaidGroups').available
+      - var pending = view.state.get('prepaidGroups').pending
+      - var anyPrepaids = available || pending
```

Comment:
Well, the view is loading when this would happen, so nothing rendered is visible anyway until the values get set to `undefined`, but it *is* preferable to handle these more explicitly. Just doing `available.length > 0` would break, though, when `available` is undefined. How about `_.size(available) > 0`, since `_.size(undefined)` resolves to 0?